c_emulator/riscv_platform_impl.c

```diff
@@ -32,6 +32,8 @@ uint64_t rv_ram_size = UINT64_C(0x4000000);
 uint64_t rv_rom_base = UINT64_C(0x1000);
 uint64_t rv_rom_size = UINT64_C(0x100);
 
+bool rv_vext_vl_use_ceil = false;
```

Comment:
The commit message doesn't mention that you're changing the default behaviour

Reply:
Sorry..., Is this commit message ok?

Reply:
I'm not sure which commit message you changed... but don't worry I will squash it and write a descriptive message.